appveyor.yml

```diff
@@ -89,17 +108,17 @@ install:
     - cmd: conda update --yes --quiet conda
 
     - cmd: conda install --yes --quiet obvious-ci conda-build-all=1.0.0
-    - cmd: conda install --yes --quiet conda-forge-build-setup
+    - cmd: conda install --yes --quiet conda-forge-build-setup=3.*
```

Comment:
Without this the build fails in the patch process. See f65966a which is using conda-forge-build-setup==4.0.0.